python/packages/autogen-core/docs/src/user-guide/core-user-guide/design-patterns/multi-agent-debate.ipynb

```diff
@@ -318,11 +318,17 @@
             ],
             "source": [
                 "runtime = SingleThreadedAgentRuntime()\n",
+                "\n",
+                "solver_a_model_client = OpenAIChatCompletionClient(model=\"gpt-4o-mini\")\n",
```

Comment:
It is okay to share model clients among agents. So we don't really need to split them apart.